pylintrc

```diff
@@ -5,9 +5,11 @@ reports=no
 # PYLINT DIRECTORY BLACKLIST.
 ignore=_vendor,_generated,samples,examples,test,tests,doc,.tox
 
-init-hook='import sys; sys.path.insert(0, os.path.abspath(os.getcwd().rsplit("azure-sdk-for-python", 1)[0] + "azure-sdk-for-python/scripts/pylint_custom_plugin"))'
+#Remove 'from regex_path_filter import RegexPathFilter; RegexPathFilter()' to run Storage
```

Comment:
Small nit maybe add a TODO to make sure this comment pops out a little more for future devs who are looking at this file?
```suggestion
# TODO remove 'from regex_path_filter import RegexPathFilter; RegexPathFilter()' to run Storage
```